templates/focus-ios.xliff

```diff
@@ -25,9 +50,9 @@
       </trans-unit>
     </body>
   </file>
-  <file original="Blockzilla/Intro.strings" source-language="en" datatype="plaintext">
+  <file original="Blockzilla/en.lproj/Intro.strings" source-language="en" datatype="plaintext">
```

Comment:
Unless I'm confused, the fact that the file name here changes will invalidate all the existing translations contained in that section (@flodolo to confirm).
I believe that's the only one file name change left to correct this time though.

Reply:
Confirmed: if the file changes, the strings are removed. You can double check any of the complete locales, their translations are lost.